### PR TITLE
Win: Test r40

### PR DESCRIPTION
**Pull request by izahn**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by izahn** [comment]
@conda-forge-admin please restart ci

**Comment by izahn** [comment]
Yeah, it produces "WARNING:conda.gateways.disk.delete:Could not remove or rename" too, but the build eventually finishes! Definitely something changed in 4.1 that causes it to get stuck.

**Comment by mbargull** [comment]
4.1 build got stuck again at the same thing, apparently. The last log output lines are always those with `\lib\R\library\base\help\&.hexmode.html`. It could be that it gets stuck on something else which is doesn't log; but it could also be that something funky is happening with those `&`-including lines. `conda-build` uses `conda`'s unlinking code and `conda` (unfortunately) uses shell/`CMD` here and there to try to be faster than deleting things from Python code. But we all now that CMD code is brittle, so it could be that something unintended happens here.
Just like Devon in https://github.com/conda-forge/r-base-feedstock/pull/173#issuecomment-844701750, I'd also think that those filenames indicate something fishy going on in R's build. But before trying to figure out what that might be, we can try to see if `conda-build --keep-old-work` makes a difference (i.e., trying to see if those `conda.gateways.disk.delete` just originate from `conda-build`'s cleanup phase) and whether we can see some other possible culprit then.

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ jobs:
     # Special cased version setting some more things!
     - script: |
         call activate base
-        conda.exe build --keep-old-work "recipe" -m .ci_support\%CONFIG%.yaml
```
Comment (mbargull) [inline .azure-pipelines/azure-pipelines-win.yml]: That didn't accomplish anything :shrug:

**Comment by mbargull** [comment]
From https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=320231&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=4179 onward we can see that if we build the HTML files on other platforms, then we get similarly problematic filenames as on Windows (though they get better handled on Linux)..
But having files with unescaped characters like `<`, `|`, etc. is rather daring, IMHO.

**Comment by izahn** [comment]
Looks like we have this figured out over in https://github.com/conda-forge/r-base-feedstock/pull/173

**Pull request closed without merging**